src/main/java/com/github/dockerjava/api/model/HostConfig.java

```diff
@@ -27,6 +27,9 @@
 	@JsonProperty("Privileged")
 	private boolean privileged;
 
+	@JsonProperty("ReadonlyRootfs")
+	private boolean readonlyRootfs;
```

Comment:
Can't this cause problems when property wouldn't exist? Boolean will default to null, while boolean may report false negative 'false'.